### PR TITLE
Setup for Bruno collection for testing access-managment API

### PR DESCRIPTION
**Pull request by jonkjetiloye**
## Description
Created a Bruno collection including:
- Environment config for all relevant environments (DEV, AT22-24, YT01, TT02 and PROD)
- .env file for secrets (see .env.sample)
- Pre-request script for getting auth-token for Personal, Enterprise and EnterpriseUser
- Import of current OpenAPI spec for Access-management
- Top-level folder split for requests for Manual Tests, and Automatic Tests
- Some initial test requests for:
-- GET maskinporten/delegations
-- POST maskinportenschema/delegationcheck
-- POST rights/delegationcheck
-- POST rights/delegation/offered

## Related Issue(s)
- #547

### CODE REVIEW

File: test/Bruno/Altinn.AccessManagement/.env.sample
Comment (andreasisnes) [inline test/Bruno/Altinn.AccessManagement/.env.sample:1]: does TOKEN_URL need to be a secret?

Comment (jonkjetiloye) [reply test/Bruno/Altinn.AccessManagement/.env.sample:1]: It's an internal tool, but not strictly a secret no.
I thought the repo was private but I see now it's public also so..

Comment (jonkjetiloye) [reply test/Bruno/Altinn.AccessManagement/.env.sample:1]: Moved the token url from .env to just a constant in the pre-request script

Comment (andreasisnes) [reply test/Bruno/Altinn.AccessManagement/.env.sample:1]: hehe, nice. It wasn't ment as a requirement from my side, I just prefer to minimize initial setup.

**Review by andreasisnes — APPROVED** [review]

**Pull request merged**